python/requirements.txt

```diff
@@ -13,4 +13,10 @@ nest_asyncio
 prompt_toolkit
 mock
 pytest-asyncio
-lastmile_utils==0.0.3
\ No newline at end of file
+lastmile_utils==0.0.3
+transformers
+
+#Pytorch
+torch
+torchvision
+torchaudio
```

Comment:
Another thing we need to keep in mind for extensibility is that some models use different versions of pytorch. Not a blocker right now but something to keep in mind since pip sucks at managing version conflicts

Reply:
Hmmm, yea that seems tough, not sure best solution for now, good to discuss more later